samples/starterKit/README.md

```diff
@@ -0,0 +1,41 @@
+# Voxa 
```

Comment:
In general i'd like to takes this as an oportunnity to make a simpler example, like, something really minimal to just show the minimum required voxa project with analytics